packages/console/src/pages/SignInExperience/components/Preview.module.scss

```diff
@@ -2,6 +2,7 @@
 
 .preview {
   width: 578px;
+  max-height: 700px;
```

Comment:
Should this be 100vh?

Reply:
100vh is too tall, and will be very ugly, consider changing size to match a mobile device in the future.